apps/site/src/_pages/docs/2_working-with-types.mdx

```diff
@@ -65,3 +65,59 @@ fetch("https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2")
 ```shell
 curl https://blockprotocol.org/@blockprotocol/types/entity-type/thing/v/2
 ```
+
+### TypeScript types for entities
+
+Types within the Type System are representable within TypeScript, and as such it is possible to create TypeScript representations of entity types.
+There is a provided `codegen` utility in the graph module package which provides an implementation of such generation.
```

Comment:
I was originally going to suggest Graph Module but I think this makes more sense
```suggestion
There is a provided `codegen` utility in the `@blockprotocol/graph` package which provides an implementation of such generation.
```